### DISCUSSION TITLE
save into a txt file

### DISCUSSION THREAD

**Discussion by fosterbrooke** [community/community#109736] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I want to save some string into a txt file in one line.
but it is saved in several lines.
help me with python code.

**Comment by devflash101** [comment]
string= string.replace('\n', ' ')
file.write(f"{string}\n")

**Reply by fosterbrooke** [reply]
i've already tried that but it didn't work.

**Reply by devflash101** [reply, marked as answer]
string = string.replace('\n', ' ').replace('\r', ' ').replace('\f', ' ').replace('\v', ' ')
file.write(f"{string}\n")
